src/vm/gdbjit.cpp

```diff
@@ -1845,12 +1845,29 @@ static inline bool isListedModule(const WCHAR *wszModuleFile)
     LPWSTR pComma = wcsstr(g_wszModuleNames, W(","));
     LPWSTR tmp = g_wszModuleNames;
 
+    // remove '.ni.dll' or '.ni.exe' suffix from wszModuleFile
```

Comment:
An idea - if you moved this to the caller of isListedModule, then you can just mutate the wszModuleFile by replacing the extension. And the compares that now have to check both the wszModuleName and wszNonNIModuleFile would check just one.